flare/agent_zoo/simple_rl_agents.py

```diff
@@ -69,3 +64,35 @@ def _cts_predict(self, observations, states):
         actions, next_states = self.predict(
             'RL', inputs=dict(sensor=observations), states=dict(state=states))
         return [actions.values()[0][0]], [next_states.values()[0][0]]
+
+
+class ActionNoiseAgent(Agent):
+    """
+    This class extends `SimpleRLAgent` by applying action noise after 
```

Comment:
Since ActionNoiseAgent is an extension of SimpleRLAgent, why not derive it from SimpleRLAgent?